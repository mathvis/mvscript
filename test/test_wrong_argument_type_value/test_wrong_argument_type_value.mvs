func a(b: int) int {
	return 0;
}
a(5.0);
