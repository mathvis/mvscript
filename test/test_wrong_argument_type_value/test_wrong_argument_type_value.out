error: Expected int but got float
 --> test/test_wrong_argument_type_value/test_wrong_argument_type_value.mvs:4:1
  |
4 | a(5.0);
  | ^
  |
hint: Try changing the argument type.
